.github/workflows/deploy-zeit-staging.yml

```diff
@@ -66,23 +66,40 @@ jobs:
           ZEIT_TOKEN: ${{ secrets.ZEIT_TOKEN }} # Passing github's secret to the worker
           CURRENT_BRANCH: ${{ github.ref }} # Passing current branch to worker
 
+      # We need to find the PR id to be able to comment it.
+      # It avoids, on many push at the same time, issues about the pull request (HEAD moving).
+      - name: Finding Pull Request ID
```

Comment:
`Finds the PR id. Will be used later to comment on that PR.`

No need to indicate the old way didn't work well here, it'll only confuse future reviewers.